src/vs/workbench/contrib/chat/browser/contrib/chatInputEditorContrib.ts

```diff
@@ -131,7 +132,23 @@ class InputEditorDecorations extends Disposable {
 	}
 }
 
-ChatWidget.CONTRIBS.push(InputEditorDecorations);
+class InputEditorSlashCommandFollowups extends Disposable {
+	constructor(
+		private readonly widget: IChatWidget,
+		private readonly chatService: IChatService
+	) {
+		super();
+		this._register(this.chatService.onDidCompleteSlashCommand((command) => this.prependSlashCommand(command)));
```

Comment:
Shouldn't this only activate for certain slash commands that need this behavior, not any slash command? 

Also I think this pattern will be triggered for _all_ chat widgets, when _any_ chat widget submits a slash command. The event could be on the widget, or, the service's event could include a session ID which you can use to check if this listener is on a widget for that session